web/src/pages/index.tsx

```diff
@@ -1,118 +1,7 @@
-import Image from "next/image";
-import { Inter } from "next/font/google";
+import { Inter } from 'next/font/google';
 
-const inter = Inter({ subsets: ["latin"] });
+const inter = Inter({ subsets: ['latin'] });
 
 export default function Home() {
-  return (
```

Comment:
This is perfect, it is correct. However, this isn't related to the issue u working on. This is a separate issue number.  Create blank page. #6 But I'll let it pass. All good.